controllers/storageoscluster/controller.go

```diff
@@ -298,6 +302,47 @@ func getCSICondition(ctx context.Context, cl client.Client, namespace string, lo
 	return csiCondition
 }
 
+func getMetricsExporterCondition(ctx context.Context, cl client.Client, namespace string, log logr.Logger) metav1.Condition {
+	metricsCondition := metav1.Condition{
+		Type:    metricsExporterReadyType,
+		Status:  metav1.ConditionFalse,
+		Reason:  notReadyReason,
+		Message: "Metrics-Exporter Not Ready",
+	}
+	nodeDS := &appsv1.DaemonSet{}
+	nodeKey := client.ObjectKey{Name: "storageos-node", Namespace: namespace}
+	if err := cl.Get(ctx, nodeKey, nodeDS); err != nil {
+		if !apierrors.IsNotFound(err) {
+			log.Error(err, "failed to get node status")
+		}
+	}
+
+	// metrics-exporter is installed only after Node
+	if nodeDS.Status.NumberReady < 1 {
```

Comment:
Why we do this, or why not we do this in others?

Reply:
Not needed, removed